lib/phoenix_test/playwright/cookies.ex

```diff
@@ -0,0 +1,74 @@
+defmodule PhoenixTest.Playwright.Cookies do
+  @moduledoc """
+  Functions to assist with adding cookies to the browser context
+
+  Note that a cookie's value must be a binary unless the cookie is signed/encrypted
+
+  ## Cookie fields
+
+  | key          | type        | description |
+  | -----------  | ----------- | ----------- |
+  | `:name`      | `binary()`  | |
+  | `:value`     | `binary()`  | |
+  | `:url`       | `binary()`  | *(optional)* either url or domain / path are required |
+  | `:domain`    | `binary()`  | *(optional)* either url or domain / path are required |
+  | `:path`      | `binary()`  | *(optional)* either url or domain / path are required |
+  | `:expires`   | `float()`   | *(optional)* Unix time in seconds. |
+  | `:http_only` | `boolean()` | *(optional)* |
+  | `:secure`    | `boolean()` | *(optional)* |
+  | `:same_site` | `binary()`  | *(optional)* one of "Strict", "Lax", "None" |
+  """
+  @type cookie :: %{
+          :name => binary(),
+          :value => binary(),
+          :url => binary(),
+          :domain => binary(),
+          :path => binary(),
+          :expires => float(),
+          :http_only => boolean(),
+          :secure => boolean(),
+          :same_site => binary()
+        }
+
+  @doc """
+  Converts the atom-keyed cookie map into a string-keyed map suitable for posting
+  """
+  def to_params_map(cookie) do
+    cookie
+    |> ensure_binary_cookie_value()
+    |> transform_to_camel_case_params_map()
```

Comment:
Probably don't need this conversion.
Should be handled by [`Playwright.Port.post/2`](https://github.com/peaceful-james/phoenix_test_playwright/blob/main/lib/phoenix_test/playwright/port.ex#L42).